core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -418,6 +418,16 @@ public T setAcl(List<AclEntry> entries) {
     return getThis();
   }
 
+  /**
+   * Sets the internal ACL to a specified ACL.
+   * @param acl the specified ACL
+   * @return the updated object
+   */
+  public T setInternalAcl(AccessControlList acl) {
```

Comment:
Why are there multiple ways to set the acl?

Reply:
because one method is setting the content of the acl and this method is actually setting the reference to acl directly.  perhaps the other one should be more appropriately named `fillAcl`?